py/release.py

```diff
@@ -407,7 +407,7 @@ def build_local(args):
 
   src_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
 
-  go_src_dir = os.path.join(go_dir, "src", REPO_ORG, REPO_NAME)
+  go_src_dir = os.path.join(go_dir, "src", "github.com", REPO_ORG, REPO_NAME)
```

Comment:
Why did you insert github.com here?

Reply:
After followed these steps
```
mkdir -p ${GOPATH}/src/github.com/kubeflow
ln -sf ${GIT_TRAINING} ${GOPATH}/src/github.com/kubeflow/tf-operator
glide install -v
go install github.com/kubeflow/tf-operator/cmd/tf_operator
cd py
pipenv install
pipenv shell
cd ..
python -m py.release local --registry=${REGISTRY}
```
There is an error.
```
INFO|2018-02-14T10:02:08|/home/jim/tf-operator/py/release.py|414| Creating symbolic link /home/jim/go/src/kubeflow/tf-operator pointing to /home/jim/tf-operator
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/jim/tf-operator/py/release.py", line 717, in <module>
    main()
  File "/home/jim/tf-operator/py/release.py", line 714, in main
    args.func(args)
  File "/home/jim/tf-operator/py/release.py", line 415, in build_local
    os.symlink(src_dir, go_src_dir)
OSError: [Errno 2] No such file or directory
```

I only found github.com under $GOPATH/src, so I add github.com.